schemas/input/csl-data.json

```diff
@@ -472,6 +482,13 @@
             "$ref": "#/definitions/name-variable"
           }
         },
+        "publisher": {
+          "description": "Publisher of the item",
+          "type": "array",
```

Comment:
Can you include annotations here, including two examples; one with a single publisher, and the other with multiple?